mobile/src/main/java/org/openhab/habdroid/core/connection/ConnectionFactory.java

```diff
@@ -100,14 +116,26 @@ public static void shutdown() {
         sInstance.mUpdateThread.quit();
     }
 
+    public static void blockingWaitForInitialization() {
+        synchronized (sInstance.mInitializationLock) {
+            while (!sInstance.mIsInitialized) {
+                try {
+                    sInstance.mInitializationLock.wait();
+                } catch (InterruptedException ignored) {
+                    // ignored
```

Comment:
do we've a checkstyle rule for that (a catch block needs a body)? If not, I would remove the comment, it's already clear from the name of the variable the exception is saved in :)

Reply:
I'm used to working with AOSP code, where such a check style rule exists. Not sure about our linter configuration.